cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/PushAttachmentsInline.java

```diff
@@ -20,10 +20,10 @@
  * </p>
  *
  * <ul>
- *     <li>False: Always push attachments separately as an HTTP binary PUT</li>
+ *     <li>False: Always push attachments separately from JSON body, via multipart/related</li>
  *     <li>Small: Push small attachments inline, and large attachments separately.
```

Comment:
Do you think we need to add a clarification here, since the behaviour is slightly different now?
I mean if I specify `Small` and have 1 large attachment my small attachments will also be sent separately now right?